modules/apps/headless/headless-admin-address/headless-admin-address-impl/src/main/java/com/liferay/headless/admin/address/internal/resource/v1_0/CountryResourceImpl.java

```diff
@@ -122,15 +122,20 @@ public Country postCountry(Country country) throws Exception {
 				ServiceContextFactory.getInstance(
 					Country.class.getName(), contextHttpServletRequest));
 
-		if (country.getTitle_i18n() == null) {
-			Map<String, String> titleMap = new HashMap<>();
+		country.setTitle_i18n(
+			() -> {
+				if (country.getTitle_i18n() == null) {
+					Map<String, String> titleMap = new HashMap<>();
 
-			for (Locale locale : _language.getAvailableLocales()) {
-				titleMap.put(_language.getLanguageId(locale), null);
-			}
+					for (Locale locale : _language.getAvailableLocales()) {
+						titleMap.put(_language.getLanguageId(locale), null);
+					}
 
-			country.setTitle_i18n(titleMap);
-		}
+					return titleMap;
+				}
+
+				return null;
```

Comment:
This means if the title_i18n had a not null value, you are setting it to null if the supplier is executed!